torch/linalg/__init__.py

```diff
@@ -483,6 +483,79 @@
     >>> tensor([ 11.6734+0.j, 105.1037+0.j,  10.1978+0.j])
 """)
 
+pinv = _add_docstr(_linalg.linalg_pinv, r"""
+linalg.pinv(input, rcond=1e-15, hermitian=False) -> Tensor
+
+Computes the pseudo-inverse (also known as the Moore-Penrose inverse) of a matrix :attr:`input`,
+or of each matrix in a batched :attr:`input`.
+The pseudo-inverse is computed using singular value decomposition (see :func:`torch.linalg.svd`) by default.
+If :attr:`hermitian` is ``True``, then :attr:`input` is assumed to be Hermitian (symmetric if real-valued),
+and the computation of the pseudo-inverse is done by obtaining the eigenvalues and eigenvectors
+(see :func:`torch.linalg.eigh`).
+The singular values (or the absolute eigenvalues when :attr:`hermitian` is ``True``) that are below
+the specified :attr:`rcond` threshold are treated to be zero and discarded in the computation.
```

Comment:
How common is the phrase "absolute eigenvalues"? Should this say "or the absolute values of the eigenvalues"?

Reply:
"are treated to be zero..." -> "are treated as zero..."

Reply:
Fixed in https://github.com/pytorch/pytorch/pull/48399/commits/ed067a80efe00e39fee6a5645e52ad8bf1946cf4